### PR TITLE
update README with section about cosign verify

### PR DESCRIPTION
**Pull request by momentmaker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,23 @@ External adapters are what make Chainlink easily extensible, providing simple in
...
 
+## Verify Official Chainlink Releases
+
+We use `cosign` with OIDC keyless signing during the [Build, Sign and Publish Chainlink](https://github.com/smartcontractkit/chainlink/actions/workflows/build-publish.yml) workflow.
+
+It is encourage for any node operator building from the official Chainlink docker image to verify the tagged release version was did indeed built from this workflow.
+
+You will need `cosign` in order to do this verification.
+
+`brew install cosign`
```
Comment (javuto) [inline README.md]: Do we want to add different instructions for systems other than macOS here?

Comment (momentmaker) [reply README.md]: good call

Comment (javuto) [reply README.md]: Pointing to the installation instructions from their website should be good: https://docs.sigstore.dev/system_config/installation/

Comment (momentmaker) [reply README.md]: yep did exactly that :)

**Review by javuto — APPROVED** [review]

**Review by vyzaldysanchez — APPROVED** [review]

**Pull request merged**